_install-and-configure/configuring-opensearch/search-settings.md

```diff
@@ -17,9 +17,9 @@ OpenSearch supports the following search settings:
 
 - `search.default_allow_partial_results` (Dynamic, Boolean):  A cluster-level setting that allows returning partial search results if a request times out or a shard fails. If a search request contains an `allow_partial_search_results` parameter, the parameter takes precedence over this setting. Default is `true`. 
 
-- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before it is canceled at the shard level. After this time has been reached, a request is stopped and all associated tasks are canceled. Default is `-1`.
+- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` interval is specified in the search request, that interval takes precedent over the configured setting. Default is `-1` (no timeout).
 
-- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1` (no timeout).
+- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before the request is canceled at the shard-level. After the specified time has been reached, the request is stopped and all associated tasks are canceled. Default is `-1`.
```

Comment:
Can we remove the text `After the specified time has been reached, the request is stopped and all associated tasks are canceled.` from here and add to documentation of `search.cancel_after_time_interval`?